lib/fluent/plugin/out_aws-elasticsearch-service.rb

```diff
@@ -20,6 +20,7 @@ class AwsElasticsearchServiceOutput < ElasticsearchOutput
       config_param :ecs_container_credentials_relative_uri, :string, :default => nil #Set with AWS_CONTAINER_CREDENTIALS_RELATIVE_URI environment variable value
       config_param :assume_role_session_name, :string, :default => "fluentd"
       config_param :assume_role_web_identity_token_file, :string, :default => nil
+      config_param :sts_credentials_region, :string, :default => nil
```

Comment:
Could you add this parameter documentation in README?
https://github.com/atomita/fluent-plugin-aws-elasticsearch-service/blob/master/README.md

Only existing in plugin code, users don't get noticed this.

Reply:
Updated an example in the README to include this config option.